src/client/article.ts

```diff
@@ -214,6 +214,45 @@ function hasSeenCards(): void {
     })
 }
 
+function safeParse(item: {} | string): any {
```

Comment:
It looks like most of the validation errors can be traced back to this `any` - I think we can get around this by changing it to `unknown`?

Reply:
Updated this PR, I thought we had handled this before...